src/coreclr/src/jit/utils.cpp

```diff
@@ -1836,6 +1847,18 @@ unsigned CountDigits(unsigned num, unsigned base /* = 10 */)
     return count;
 }
 
+unsigned CountDigits(float num, unsigned base /* = 10 */)
```

Comment:
This function seems semantically dubious. What about the fractional component? If the caller doesn't care, they should cast to unsigned and call the unsigned version.

Reply:
We already anticipate printing up to 2 fractional digits because of our old "fixed point" scheme.

Was trying to keep formatting comparable because there are lots of computed printf layouts in the jit I'd rather not mess with just yet.